tests/glommio_ct.rs

```diff
@@ -0,0 +1,85 @@
+#![ cfg( feature = "glommio" ) ]
+
+// ✔ pass a     ThreadPool  to a function that takes exec: `impl SpawnHandle`
```

Comment:
Looks like you took the tests from `ThreadPool`. This does not test any of the `LocalSpawn` and `LocalSpawnHandle` functionality. Actually the right one to copy from is `TokioCt`. I suggested `LocalPool` before because it doesn't require a builder, but it has less tests and the builder actually is only one line of code, so it will be the same to adapt.